### PR TITLE
Add missing asterik + add "or later" to vs 2017

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/core/tutorials/with-visual-studio.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ This topic provides a step-by-step introduction to building, debugging, and publ
...
+[Visual Studio 2017 or later](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 2.1 or later versions.
 
 For more information, see the [Prerequisites for .NET Core on Windows](../windows-prerequisites.md) topic.
 
 ## A simple Hello World application
 
 Begin by creating a simple "Hello World" console application. Follow these steps:
 
-1. Launch Visual Studio 2017. Select **File** > **New** > **Project** from the menu bar. In the *New Project** dialog, select the **Visual C#** node followed by the **.NET Core** node. Then select the **Console App (.NET Core)** project template. In the **Name** text box, type "HelloWorld". Select the **OK** button.
+1. Launch Visual Studio 2017. Select **File** > **New** > **Project** from the menu bar. In the **New Project** dialog, select the **Visual C#** node followed by the **.NET Core** node. Then select the **Console App (.NET Core)** project template. In the **Name** text box, type "HelloWorld". Select the **OK** button.
```
Comment (mairaw) [inline docs/core/tutorials/with-visual-studio.md]: Let's remove the version from this sentence then

```suggestion
1. Launch Visual Studio. Select **File** > **New** > **Project** from the menu bar. In the **New Project** dialog, select the **Visual C#** node followed by the **.NET Core** node. Then select the **Console App (.NET Core)** project template. In the **Name** text box, type "HelloWorld". Select the **OK** button.
```

File: docs/core/tutorials/with-visual-studio.md [hunk 2]
```diff
@@ -12,15 +12,15 @@ This topic provides a step-by-step introduction to building, debugging, and publ
 
 ## Prerequisites
 
-[Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 2.1 or later versions.
+[Visual Studio 2017 or later](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 2.1 or later versions.
```
Comment (mairaw) [inline docs/core/tutorials/with-visual-studio.md]: we shouldn't use the aka link in this case 

```suggestion
[Visual Studio 2017 or later](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 2.1 or later versions.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Youssef1313 for the changes. This article is actually on my backlog to upgrade to VS 2019 and .NET Core 3.0 but I've suggested some changes for now.

**Comment by Youssef1313** [comment]
@mairaw, I've made some additional minor changes that need to be reviewed.

**Review by mairaw — APPROVED** [review]
LGTM. Thanks @Youssef1313!

**Pull request merged**